lms/templates/courseware/progress.html

```diff
@@ -180,12 +180,30 @@ <h4 class="hd hd-4">
                                             %endif
                                         </p>
                                         %if len(section.problem_scores.values()) > 0:
-                                        <dl class="scores">
-                                            <dt class="hd hd-6">${ _("Problem Scores: ") if section.graded else _("Practice Scores: ")}</dt>
-                                            %for score in section.problem_scores.values():
-                                            <dd>${"{0:.3n}/{1:.3n}".format(float(score.earned),float(score.possible))}</dd>
-                                            %endfor
-                                        </dl>
+                                          %if section.show_grades(staff_access):
+                                          <dl class="scores">
```

Comment:
@yro (Taking over for @pomegranited since she's away.)

IIUC, you're asking if it would be possible to do

```
%if staff_access:
```

instead of

```
%if section.show_grades(staff_access):
```

here.

Doing that wouldn't give us the behavior that we want: If a subsection is set up to never show correctness, we want to [hide individual scores from all users](https://github.com/edx/edx-platform/pull/15018/files#diff-743e6bcf9ca89e1c3a85c3ec2c9d81b5R488), including those that have staff access, cf. https://github.com/edx/edx-platform/pull/15018#issuecomment-302102287.

This behavior mimics what we're doing for the "Show answer" setting (see the [definition](https://github.com/edx/edx-platform/pull/15018/files#diff-0c9995bbce7571d2d55664680e485452L892) of `CapaMixin.answer_available`), and was approved in #14737 ([comment](https://github.com/edx/edx-platform/pull/14737/files#r110754178)), the PR that first introduced the "Show correctness" setting.